cunumeric/fft/fft.py

```diff
@@ -860,17 +937,25 @@ def hfft(a, n=None, axis=-1, norm=None):
     Single GPU
     """
     s = (n,) if n is not None else None
-    axis = (axis,) if axis is not None else None
+    computed_axis = (axis,) if axis is not None else None
```

Comment:
Just noting I made similar changes throughout. In general, whenever there is a "convenience" arg that can accept a variety of types, to be internally adapted to single type, it's better to use a new variable to hold the computed value. (e.g. otherwise any functions that receive the value would have to accept the "wide" type specification, and/or mypy will need annoying explicit type annotations on the variable)

Reply:
I'll make a note about this in the weekly meeting next week.